gatekeeperv2config.json

```diff
@@ -0,0 +1,143 @@
+    "FieldName": "guildID",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "guildID",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder": "883220743755206666"
+  },
+  {
+    "DisplayName": "AMP User",
+    "Category": "Bot Settings",
+    "Description": "The Login Name for the account you want the bot to use. This account must have \"Super Admin\" privelege for setup of new instances and after upgrades.",
+    "Keywords": "amp,user",
+    "FieldName": "AMPUser",
+    "InputType": "text",
+    "IsFlagArgument": false,
+    "ParamFieldName": "AMPUser",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder": "amp-bot"
+  },
+  {
+    "DisplayName": "AMP Password",
+    "Category": "Bot Settings",
+    "Description": "The password for said Login.",
+    "Keywords": "amp,password",
+    "FieldName": "AMPPassword",
+    "InputType": "password",
+    "IsFlagArgument": false,
+    "ParamFieldName": "AMPPassword",
+    "IncludeInCommandLine": false,
+    "DefaultValue": ""
+  },
+  {
+    "DisplayName": "AMP 2FA Token",
+    "Category": "Bot Settings",
+    "Description": "The 2Factor AUTH Code for said Login.",
+    "Keywords": "amp,2factor,2fa,token",
+    "FieldName": "AMPAuth",
+    "InputType": "password",
+    "IsFlagArgument": false,
+    "ParamFieldName": "AMPAuth",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder": "MQ2TMMETGB4WCYJY8NRDINDEMKQTGMZTMIZDOZTFME3TSMRXGA2A"
+  },
+  {
+    "DisplayName": "AMP URL",
+    "Category": "Bot Settings",
+    "Description": "The URL or IP of your AMP ADS instance.",
+    "Keywords": "amp,url",
+    "FieldName": "AMPurl",
+    "InputType": "text",
+    "IsFlagArgument": false,
+    "ParamFieldName": "AMPurl",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "http://127.0.0.1:8080",
+    "Placeholder": "http://127.0.0.1:8080"
+  },
+  {
+    "DisplayName": "Super Admin",
+    "Category": "Bot Settings",
+    "Description": "The bot requires Super Admin for first time setup of new instances and after upgrades. Enable this to keep the permission intact. Use at your own risk as this poses some security risk at the expense of convenience.",
+    "Keywords": "super,admin",
+    "FieldName": "super",
+    "InputType": "checkbox",
+    "IsFlagArgument": false,
+    "ParamFieldName": "super",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {
+        "False": "",
+        "True": " -super"
+    }
+  },
+  {
+    "DisplayName": "Disable Discord Integration",
+    "Category": "Bot Settings",
+    "Description": "This setting is for dev use.",
+    "Keywords": "discord,integration",
+    "FieldName": "discord",
+    "InputType": "checkbox",
+    "IsFlagArgument": false,
+    "ParamFieldName": "discord",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "EnumValues": {
+        "False": "",
+        "True": " -discord"
```

Comment:
So if this param is used, Discord integration is disabled?

Reply:
Yes, it was intended for development purpose.

Reply:
I would have preferred to call this one `Discord Integration` and default it to True. But AMP doesn't respect the default value there. So I swapped them and called it a day.

Reply:
My question was whether when it is true (the discord param is used) it is actually disabled, or instead enabled?

Reply:
Sorry, I thought the explanation answered that. The parameter disables Discord integration. So this one would be reversed from the others settings like it. Since AMP doesn't like a blank default value and flag arguments don't work... It ended up this way.

Reply:
👍